dpdata/plugins/ase.py

```diff
@@ -296,6 +298,24 @@ def from_labeled_system(
 
         return dict_frames
 
+    def to_system(self, data, **kwargs):
+        """Convert System to ASE Atoms object."""
+        list_atoms = ASEStructureFormat().to_system(data, **kwargs)
+        file_name = kwargs.get("file_name", "conf.traj")
```

Comment:
Could you explicitly write the argument in line 301 - the current code cannot generate docs

Reply:
done